### PR TITLE
Fix transfers

### PR DESCRIPTION
**Pull request by exu3**
It seems like I broke disbursements in #2373 

Cases to test:

- [x] Making a disbursement -- sorta, the new_event_disbursement_path is still broken
- [x] Making a disbursement of an amount higher than the source event balance
- [x] Making an ACH
- [x] ACH of amount higher than source event balance
- [ ] Making a check - errrrr I think this is broken, not sure why -- seems not to be related to this PR

### CODE REVIEW

**Comment by maxwofford** [comment]
Thanks for pushing this! Testing now

File: app/services/disbursement_service/create.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def initialize(source_event_id:, destination_event_id:,
     def run
       raise ArgumentError, "amount is required" unless @amount
       raise ArgumentError, "amount_cents must be greater than 0" unless amount_cents > 0
-      raise ArgumentError, "You don't have enough money to make this disbursement." unless ample_balance?(amount_cents, @source_event)
+      raise ArgumentError, "You don't have enough money to make this disbursement." unless ample_balance?(amount_cents, source_event)
```
Comment (maxwofford) [inline app/services/disbursement_service/create.rb]:
```suggestion
      raise ArgumentError, "You don't have enough money to make this disbursement." unless ample_balance?(amount_cents, @source_event)
```

File: app/controllers/disbursements_controller.rb [hunk 2]
```diff
@@ -60,7 +60,7 @@ def create
 
   rescue ArgumentError, ActiveRecord::RecordInvalid => e
     flash[:error] = e.message
-    redirect_to new_event_disbursement_path(event_id: @source_event&.slug)
+    redirect_to new_disbursement_path(event_id: @source_event.id)
```
Comment (maxwofford) [inline app/controllers/disbursements_controller.rb]:
```suggestion
    redirect_to new_event_disbursement_path(event_id: disbursement_params[:source_event_id])
```

I think this is the intent of this change?

**Comment by exu3** [comment]
These changes were approved by Max over Slack

**Pull request merged**